bitcoin/BRWalletManager.c

```diff
@@ -680,6 +688,7 @@ BRWalletManagerStart (BRWalletManager manager) {
 
 extern void
 BRWalletManagerStop (BRWalletManager manager) {
+    BRSyncManagerDisconnect (manager->syncManager);
```

Comment:
Not a call to `BRWalletManagerDisconnect()`?  [If disconnecting involves more than just `BRSyncManagerDisconnect()`, you'd then be duplicating it.]

Reply:
There is no `bwm->state` thingy to track?

Reply:
No `bwm->state`. I'm not a fan of each layer tracking information that is more accurate in a layer below it.

Reply:
^^^?